recipes-nymea/libnymea-gpio/libnymea-gpio.bb

```diff
@@ -0,0 +1,14 @@
+DESCRIPTION = "libnymea-gpio package"
+
+LICENSE = "CLOSED"
```

Comment:
It's not closed. libnymea-gpio is LGPL-3.0 | NYMEA-COMMERCIAL.

Also, technically this repository has a lib but also a tool. I would personally name the package nymea-gpio. Then have two packages, one for the lib, one for the tool.

Each will need its own license because the tools are GPL-3.0 | NYMEA-COMMERCIAL.

Reply:
Thanks for taking a look @QSchulz , I will fix this!

Reply:
Renamed the recipe and split the tool into an additional package (with a different license).